packages/calling/src/Metrics/types.ts

```diff
@@ -47,6 +52,13 @@ export interface IMetricManager {
     type: METRIC_TYPE,
     error: LineError | CallingClientError | undefined
   ) => void;
+  submitBNRMetric: (
+    name: METRIC_EVENT,
+    metricAction: string,
```

Comment:
Shouldn't this be the enum we've created above?

Reply:
For submitCallMteric and submitVoicemailmetric, it was using string as a type so I just used the same format. Updated the code now